### PR TITLE
Remove Unused Notifications

### LINKED ISSUE #894: Remove Unused Notifications
- [x] Invest: powerUp, powerDown
- [x] Wallet: transfer, receive NTZ, ETH
- [x] Exchange: purchase, sell NTZ
- [x] Account-related

### PR DESCRIPTION
**Pull request by VonIobro**
- [x] Resolves #894 
Tested game and dashboard updates on Firefox and Chrome
@sunify Ready to review and merge

### CODE REVIEW

**Comment by VonIobro** [comment]
@sunify Ready to merge^^

File: app/containers/Dashboard/txnsToList.js [hunk 1]
```diff
@@ -6,16 +6,15 @@ import BigNumber from 'bignumber.js';
...
 import { Icon, TypeIcon, Error, ErrorIcon, typeIcons } from './styles';
 import messages from './messages';
 import {
   isSellEvent,
   isETHPayoutEvent,
-  isABPPayoutEvent,
-  isPurchaseStartEvent, isPurchaseEndEvent,
-  isPowerUpEvent,
+  isPurchaseStartEvent,
+  isPurchaseEndEvent,
```
Comment (sunify) [inline app/containers/Dashboard/txnsToList.js]: Purchase/sell stuff also can be safely removed

File: app/containers/Notifications/constants.js [hunk 2]
```diff
@@ -2,14 +2,9 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import A from 'components/A';
 import { Icon } from 'containers/Dashboard/styles';
-import Link from '../../components/Link';
 import { conf } from '../../app.config';
 
-export const TRANSFER_NTZ = 'transfer_ntz';
-export const TRANSFER_ETH = 'transfer_eth';
-export const SELL_NTZ = 'sell_ntz';
 export const ETH_PAYOUT = 'eth_payout';
```
Comment (sunify) [inline app/containers/Notifications/constants.js]: And payout stuff too

File: app/containers/App/sagas/index.js [hunk 3]
```diff
@@ -1,15 +1,11 @@
...
+import { fork } from 'redux-saga/effects';
 
 import gtmSaga from './gtmSagas';
-import unsupportedModalSaga from './unsupportedModalSaga';
 import balancesLoadingSaga from './balancesLoadingSaga';
 
 export function* appSaga(dispatch) {
   yield fork(gtmSaga);
   yield fork(balancesLoadingSaga, dispatch);
-  yield takeEvery(ACCOUNT_LOADED, unsupportedModalSaga);
```
Comment (sunify) [inline app/containers/App/sagas/index.js]: `unsupportedModalSaga.js` can be removed as long as we don't need MetaMask

File: app/containers/Dashboard/txnsToList.js (hunk 1 above)
Comment (VonIobro) [reply app/containers/Dashboard/txnsToList.js]: ok, thanks for the tips^^

File: app/containers/Dashboard/txnsToList.js [hunk 4]
```diff
@@ -115,38 +111,18 @@ function infoIcon(event) {
...
     return <FormattedMessage key="descr" {...messages.ethPayoutStatus} />;
   } else if (isSellEvent(event)) {
     return <FormattedMessage key="descr" {...messages.sellStatus} />;
-  } else if (isPurchaseEndEvent(event, address)) {
-    return <FormattedMessage key="descr" {...messages.purchaseEnd} />;
-  } else if (isPurchaseStartEvent(event)) {
-    return <FormattedMessage key="descr" {...messages.purchaseStart} />;
   }
 
   return <FormattedMessage key="descr" {...messages.transferStatus} />;
```
Comment (VonIobro) [inline app/containers/Dashboard/txnsToList.js]: @sunify Can `ethPayoutStatus`, `sellStatus`, and `transferStatus` be removed from txnsToList.js?

Comment (sunify) [reply app/containers/Dashboard/txnsToList.js]: Yep

**Pull request merged**